linux/deploy_vm/rebuild_pardus_unattend_install_iso.yml

```diff
@@ -28,23 +40,28 @@
   ansible.builtin.replace:
     path: "{{ src_iso_file_dir }}/isolinux.cfg"
     regexp: "default .*"
-    replace: "default installgui"
+    replace: "default autoinstall"
 
 - name: "Update default boot label"
   ansible.builtin.lineinfile:
     path: "{{ src_iso_file_dir }}/{{ item }}"
-    insertbefore: "^label install$"
-    line: "default installgui"
+    firstmatch: true
+    insertbefore: "^label install*"
+    line: "default autoinstall"
   with_items:
     - "install_en.cfg"
     - "install.cfg"
 
-# Add DPMS=-s\ 0 to boot command line to disable screen saver for debian installer
-- name: "Update boot parameters with preseed.cfg"
-  ansible.builtin.replace:
+- name: "Add menu entry for autoinstall"
+  ansible.builtin.blockinfile:
     path: "{{ src_iso_file_dir }}/{{ item }}"
-    regexp: 'append'
-    replace: 'append auto=true file=/cdrom/{{ unattend_install_file_name }} "DPMS=-s 0"'
+    insertafter: "^default autoinstall$"
+    block: |
+      label autoinstall
+          menu label ^Automated Graphical Installer
```

Comment:
I think we can remove 'Graphical' now, as it might be text installer for Pardus server.